deel-1-flexbox-positioneren/styles/styles.css

```diff
@@ -1 +1,78 @@
-/*Schrijf hier jouw CSS*/
\ No newline at end of file
+/*Schrijf hier jouw CSS*/
+
+.opdracht-1 ul {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    align-content: center;
+}
+
+.opdracht-2 ul {
+    display: flex;
+    flex-direction: row;
+}
+
+.opdracht-2 ul li {
+    flex-basis: 300px;
+}
+
+.opdracht-3 {
+    display: flex;
+    align-items: stretch;
+}
+
+.opdracht-3 ul {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    flex-basis: 796px;
+}
+
+.opdracht-3-1 {
+    align-self: flex-start;
+}
+
+.opdracht-3-2 {
+    align-self: center;
+}
+
+.opdracht-3-3 {
+    align-self: flex-end;
+}
+
+.opdracht-4 ul {
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+    gap: 10px;
+}
+
+.opdracht-5 {
```

Comment:
in de randvoorwaarden aan het begin (paragraaf 1.2) staat dat we de section elementen niet mochten gebruiken om elementen op uit te lijnen ... maar ik moet eerlijk zeggen dat het mij ook niet gelukt is zonder de section te gebruiken of zonder width te gebruiken in de div wat volgens mij ook niet mocht